src/integrationTest/java/org/radarcns/integration/E4AggregatedAccelerationMonitor.java

```diff
@@ -75,11 +74,11 @@ protected void evaluateRecords(ConsumerRecords<GenericRecord, GenericRecord> rec
                 return;
             }
             GenericRecord value = record.value();
-            GenericData.Array count = (GenericData.Array) value.get("count");
+            GenericData.Array fields = (GenericData.Array) value.get("fields");
             logger.info("Received [{}, {}, {}] E4 messages",
-                    count.get(0), count.get(1), count.get(2));
+                    ((GenericRecord)fields.get(0)).get("count"), ((GenericRecord)fields.get(1)).get("count"), ((GenericRecord)fields.get(2)).get("count"));
 
-            if ((Double)count.get(0) > 200) {
+            if ((Integer)((GenericRecord)fields.get(0)).get("count") > 100) {
```

Comment:
What exactly is the number `100`?  Why do we check whether we consumed more than 100 records?

Reply:
It's a bit arbitrary... Generally, that we get more than 100 records over a 10 second interval.